financials/spending_log.csv

```diff
@@ -1,4 +1,6 @@
 spend_id,date,spend_description,decision_log_id,currency,amount,exchange_rate,USD_equivalent,from_address,to_address,tx_reference,comment
 1,20190301,Coinspect audit fee,19,BTC,4.482,3882.66,17402.08,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,17Du5PQfcLq1yeV8yw512M5UnkeZvTwe7f,https://www.blockchain.com/btc/tx/022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468,30% advance payment
 2,20190314,Donation to Ignotus Peverell for 3 months worth of work,27,GRIN,9593.296153967,2.70,25901.9,https://donation.grin-tech.org,-,1ed0e7c7-6f15-4ed6-97f2-b1c16a6af7f3,not enough grins for full 30k payment so rest to be completed when possible
-3,20190304,Discourse forum operation costs,-,USD,3600,1,3600,-,-,-,As discussed and agreed between Igno Peverell and Blockcypher 
+3,20190304,Discourse forum operation costs,-,USD,3600,1,3600,-,-,-,As discussed and agreed between Igno Peverell and Blockcypher
+4,20190412,Donation to Lehnberg,31,ETH,54.66472303,164.64,9000,0x16726940ecebdfb3c352c29bf2620f59ef919a3b,0x9471b084129b4974A1a0920EB39a1621966E3aF0,0x90f31886fd8a7cbaa6dce2bf2b935a12252983a705508597ce2931bb44d64b09, Combined with ETH/BTC conversion for the dev fund

```

Comment:
Mention that it's for 6 months?

Reply:
Thanks for the review @kargakis - The entry references `decision_log_id` 31: https://github.com/mimblewimble/grin-pm/blob/master/notes/20190409-meeting-governance.md#decision-lehnberg-funding which has the exact details. 

I prefer to avoid entering the same data in multiple places as it tends to be error prone and confusing, in particular if/when things change.

In my opinion, the objective of the log is to only track spends, and the particular circumstances of the transaction, not the motivation or conditions of a spend, which should be captured elsewhere.

Reply:
I brought this up to stay consistent with Igno's payment but I also think mentioning the duration of a payment, when we know that info, is an important part of a spend.

Reply:
Not meant to block this PR, can be fixed in a follow-up (would also have to fix the forum payment).